tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -455,4 +455,20 @@ class HoverTermSuite extends BaseHoverSuite {
       "3" -> "case CONTINUE: FileVisitResult".hover
     )
   )
+
+  check(
+    "toplevel".tag(IgnoreScala2),
+    """
+      |val (first, <<se@@cond>>) = (1, false)
+      |""".stripMargin,
+    """|```scala
+       |package 
+       |```
+       |""".stripMargin,
```

Comment:
Could you align pipes? That offset really bothers me 😆

Reply:
Fixed!